doc/ref_impl/cntt-ri2/chapters/chapter02.md

```diff
@@ -22,10 +22,20 @@ interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 ## 2.2 Reference Architecture Specification
 
-| RA2 Section | RA2 Reference  | Description | Requirement for Basic Profile | Requirement for Network Intensive Profile| RI2 Traceability |
-|---|---|---|---|---|---|
-|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|Not required|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|_TBC_|
-|||||||
+| RA2 Section | RA2 Reference  | Specification | Requirement | Requirement for Basic Profile | Requirement for Network Intensive Profile | RI2 Traceability |
+|---|---|---|---|---|---|---|
+|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|Must support|Not required|[4.3.1](./chapter04.md#431-installation-on-bare-metal-infratructure)|
```

Comment:
Typo: The Basic and Network Intensive values have been interchanged
```suggestion
|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|Not required|Must support|[4.3.1](./chapter04.md#431-installation-on-bare-metal-infratructure)|
```

Reply:
That was definitely incorrect. However, looking at the RA2 `infra.com.cfg.004` it mentions Must/Must, while the RM and the RA2 `i.cap.018` only requires it for the network intensive profile.
I've aligned with the "Kubernetes Node" requirement for now (Must/Must), even though it is likely only a must for network intensive.